cv_bridge/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 cmake_minimum_required(VERSION 2.8)
 project(cv_bridge)
 
+add_definitions(-std=c++11)
```

Comment:
Actually I would to this differently. E.g. like this: https://github.com/ethz-asl/oomact/blob/master/oomact/CMakeLists.txt#L9-L14 
(never mind the `-std=c++11` in the line above the ones I've marked. It is not necessary for this library)